prac_06/car.py

```diff
@@ -0,0 +1,36 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, name="", fuel=0):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.name = name
+        self.fuel = fuel
+        self._odometer = 0
+
+    def __str__(self):
+        """Return a string representation of car object."""
+        return f"{self.name}, {self.fuel}, {self._odometer}"
+
+    def add_fuel(self, amount):
+        """Add amount to the car's fuel."""
+        self.fuel += amount
+
```

Comment:
May be useful to add a parameter to only allow positive amounts of fuel to be added

Reply:
That's not the class's job. Can you think of any error-checking we've done in class methods?